cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/ConnectCluster.java

```diff
@@ -95,7 +99,7 @@ public void shutdown() {
      * @return      Port which is used by given Connect node
      */
     public int getPort(int node) {
-        return connectInstances.get(node).adminUrl().getPort();
+        return connectPorts.get(node);
```

Comment:
is this supposed to stay like this? or that's just a temp hack? since connectedPorts seems to be a list of ports

Reply:
I did not plan any further changes to this. `adminUrl` method was removed from Kafka, the new code does the same, just in a different way.